easybuild/easyconfigs/h/h5py/h5py-2.10.0-intel-2020a-Python-2.7.18.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonPackage'
+
+name = 'h5py'
+version = '2.10.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.h5py.org/'
+description = """HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,
+ version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous
+ amounts of data."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCE_TAR_GZ]
+patches = ['h5py-%(version)s_avoid-mpi-init.patch']
+checksums = [
+    '84412798925dc870ffd7107f045d7659e60f5d46d1c70c700375248bf6bf512d',  # h5py-2.10.0.tar.gz
+    '6bacb71f5d9fbd7bd9a01018d7fe21b067a2317f33c4a7c21fde9cd404c1603f',  # h5py-2.10.0_avoid-mpi-init.patch
+]
+
+#builddependencies = [('pkgconfig', '1.5.1', versionsuffix)]
```

Comment:
leftovers here

Reply:
Yeah, indeed. I had to comment this out because `pkgconfig` requires `poetry` as a build dep, and things get *very* messy there with Python 2. Not only do I have to resort to old versions of a bunch of required Python packages which dropped compatibility with Python 2 recently, but I also hit a requirement for `dbus` (via `keyring` -> old version of `SecretStorage` I think), which requires `dbus` to be installed in the OS.

The installation works fine without `pkgconfig` as a build dep though, which should be good enough for this very specific use case.